src/components/20-molecules/accordeon/index.scss

```diff
@@ -0,0 +1,86 @@
+.m-accordeon {
+  display: block;
+  max-width: 450px; //temporary
```

Comment:
This will be discussed with a designer, so ignore this for the moment

Reply:
When you have stuff like this, it is a good practice to add an  `@todo` into the comment. IDEs can recognize that and warn you, if you would like to commit unfinished stuff and it's easier to find parts in the code that actually need some love.